harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
It seems like that test has three lines, each testing one field of the descriptor. verifyConfigurable should go last in that case.